speech/google/cloud/speech/_gax.py

```diff
@@ -43,15 +43,22 @@ class GAPICSpeechAPI(object):
 
     :type client: `~google.cloud.core.client.Client`
     :param client: Instance of ``Client`.
+
+    :type client_config: dict
+    :param client_config: A dictionary for call options for each method. See
+                :func:`google.gax.construct_settings` for the structure of
+                this data. Falls back to the default config if not specified
+                or the specified config is missing data points.
```

Comment:
@lukesneeringer It's been our practice to either align text or indent all of it. So

```python
"""
    :param client_config: A dictionary for call options for each method. See
                          :func:`google.gax.construct_settings` for the
                          structure of this data. Falls back to the default
                          config if not specified or the specified config is
                          missing data points.
"""
```

or

```python
"""
    :param client_config:
        A dictionary for call options for each method. See
        :func:`google.gax.construct_settings` for the structure of
        this data. Falls back to the default config if not specified
        or the specified config is missing data points.
"""
```

However we don't have any lint rule enforcing it, so anything goes?